content/02.introduction.md

```diff
@@ -157,14 +157,17 @@ This excessive clotting in lung capillaries has been suggested to related to a d
 Individual outcomes have varied significantly, and reported hospitalization rates have been wide-ranging based on age and location, with various underlying health conditions being disproportionately reported [@doi:10.15585/mmwr.mm6915e3].
 Outcomes in patients that are hospitalized have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
 
-Though older populations are generally considered the most vulnerable, pediatric infections are also a concern.
+The results of COVID-19 infection can vary greatly among pediatric patients as well as adults. 
```

Comment:
I think that is a good improvement, although it might be better to say "and perhaps can" instead of "and can," because we don't know for sure that the MIS/C syndrome doesn't happen in adults too.